handlers/validator.go

```diff
@@ -130,8 +135,13 @@ func Validator(w http.ResponseWriter, r *http.Request) {
 			var name string
 			err := db.ReaderDb.Get(&name, `SELECT name FROM validator_names WHERE publickey = $1`, pubKey)
 			if err != nil && err != sql.ErrNoRows {
-				logger.Errorf("error getting validator-name from db for pubKey %v: %v", pubKey, err)
-				http.Error(w, "Internal server error", http.StatusInternalServerError)
```

Comment:
We should only come here if the Database has an error (like not reachable or so) and then its ok to show the error.